api/hooks/driver.js

```diff
@@ -39,10 +39,7 @@ function Driver(sails) {
      * @param {Function} callback The initialize callback
      */
     initialize(callback) {
-      sails.after([
-        'hook:config:loaded',
-        'hook:orm:loaded'
-      ], () => {
+      sails.after('hook:config:loaded', () => {
```

Comment:
orm needs to be loaded because `MachineService` is responsible for creating the image. See https://github.com/Nanocloud/nanocloud/blob/master/api/services/MachineService.js#L110